II.pass-tests/05-factorial/factorial.js

```diff
@@ -9,7 +9,24 @@
 // note: factorial of 0 is 1
 
 function factorial(int) {
+        if (int <= 1) {
+            return int
+        } else {
+            return int * factorial(int - 1)
+        }
+    }
 
+    factorial(5)
+
+function factorial(int) {
```

Comment:
is it necessary to have two functions defined? Both are called `factorial` and JS will likely interpret the last one, ignoring the first.
We are also making two calls `factorial(5)` and `factorial(1000)` but do we need them?

Reply:
@lpalma thanks for that observation, will make the correction